src/lib/php/Report/LicenseMainGetter.php

```diff
@@ -41,7 +41,7 @@ public function __construct()
     parent::__construct($groupBy = 'text');
   }
 
-  protected function getStatements($uploadId, $uploadTreeTableName, $groupId = null)
+  protected function getStatements($uploadId, $uploadTreeTableName, $groupId = null, $objectAgent)
```

Comment:
Please update the abstract function `getStatements` declared in `ClearedGetterCommon` to match the signature.